CHANGES.md

```diff
@@ -24,17 +24,29 @@ Important changes:
  * feat: switch to using wordnet instead of spell for finding English words in spell.sh (Max Leske) [#3242]
  * fix: refactor and rename anomaly scoring variables and paranoia level definition (Simon Studer) [#2417]
  * update: drop HTTP/0.9 support to resolve FP (Federico G. Schwindt) [#1966]
+ * update: nodejs to generic category (Felipe Zipitría) [#2340]
+ * tests: 100% test coverage for rules (Entire team, Juan-Pablo Tosso, NiceYouKnow)
 
-Tool changes:
+Tool and container changes:
```

Comment:
I wouldn't change the name here. The only thing related to containers is our `docker-compose.yaml`, which we use for testing. IMO, that should go into the same category as changes to tests go.

Reply:
I'm fine with that. Made that change.

When I renamed the section, I was not sure how much more I would be getting.